consensus/config/src/parameters.rs

```diff
@@ -68,6 +68,11 @@ pub struct Parameters {
     /// Tonic network settings.
     #[serde(default = "TonicParameters::default")]
     pub tonic: TonicParameters,
+
+    /// Time to wait during node start up until the node has synced the last proposed block via the
+    /// network peers. When set to `0` the sync mechanism is disabled.
+    #[serde(default = "Parameters::default_sync_last_proposed_block_timeout")]
+    pub sync_last_proposed_block_timeout: Duration,
```

Comment:
See PR's description for the logic behind disabling the mechanism when set to `0`